.github/workflows/deploy.yml

```diff
@@ -9,6 +9,12 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - uses: mstksg/setup-stack@v1
+    - uses: actions/cache@v1
+      with:
+        path: ~/.stack
+        key: dot-stack-${{ hashFiles('docs/**/*.md') }}
```

Comment:
I'm a bit confused by this. Normally the cache key should be derived from something that will actually affect the contents of the cache. Do you expect a change in the markdown files to affect the contents of `~/.stack`?

Reply:
You are right, but then the only sensible thing that can go there is `stack.yaml`! 😉